dask/dataframe/io/parquet/core.py

```diff
@@ -1414,6 +1427,12 @@ def apply_conjunction(parts, statistics, conjunction):
                 out_parts.append(part)
                 out_statistics.append(stats)
 
+    if _missing_stats:
+        warnings.warn(
+            f"Columns {_missing_stats} are missing min/max statistics. "
+            f"Do not expect effective predicate pushdown!"
+        )
+
```

Comment:
This feels a bit heavy handed. Maybe something in the docs would be sufficient. If we're going to emit a warning, it should be a `PerformanceWarning` not a `UserWarning` I think

Reply:
I don't feel strongly about raising a warning or being silent. However, I have fielded several complaints about Dask silently ignoring user-provided `filters` and/or `calculate_divisions` arguments when there are missing statistics. I'd be happy to remove this warning and revisit it more carefully later.

Reply:
Oh, does `filters=` not work if there are no statistics? If that's the case, we might want to just raise an error.

>  I'd be happy to remove this warning and revisit it more carefully later.

Let's do that for now. We might ultimately end up adding the same thing back in. But it's separate from the original fix here, so I'd prefer to move it to a separate PR if you're okay with it

Reply:
>Oh, does filters= not work if there are no statistics? If that's the case, we might want to just raise an error.

Right, exactly. However, given that we *may* have a predicate-pushdown optimization in the future, raising an error when `filters` can't be used seems a bit heavy-handed.

>Let's do that for now.

Works for me.